### PR TITLE
povray: Bump to 3.7.0.8

### LINKED ISSUE #3297: Povray crashes on x86_64
Attaching debug report

### PR DESCRIPTION
**Pull request by kenmays**
Closes #3297
Ref: #3858

### CODE REVIEW

File: media-gfx/povray/povray-3.7.0.8.recipe [hunk 1]
```diff
@@ -77,14 +76,24 @@ BUILD()
...
+
+	if [ $effectiveTargetArchitecture = x86_gcc2 ]; then
+		BoostLibLctr=$BoostLibLctr/x86
+		KeepEXR="--without-openexr"
+	fi
 
 	runConfigure ./configure \
 		COMPILED_BY="HaikuPorts" \
-		--with-boost-libdir=`finddir B_SYSTEM_LIB_DIRECTORY`
+		--disable-optimiz \
```
Comment (waddlesplash) [inline media-gfx/povray/povray-3.7.0.8.recipe]: What's this?

Comment (kenmays) [reply media-gfx/povray/povray-3.7.0.8.recipe]: Disables its default -march=native setting.

File: media-gfx/povray/povray-3.7.0.8.recipe [hunk 2]
```diff
@@ -77,14 +77,24 @@ BUILD()
...
+	autoupdate
 	cd ..
 
-	export CXXFLAGS="-std=c++11 -DBOOST_NO_CXX11_CONSTEXPR -D_BSD_SOURCE"
 	export LDFLAGS="-lbsd -lnetwork"
+	export CXXFLAGS="-std=c++11 -DBOOST_NO_CXX11_CONSTEXPR -D_BSD_SOURCE"
+	BoostLibLctr=$libDir
+	KeepEXR=""
+
+	if [ $effectiveTargetArchitecture = x86_gcc2 ]; then
```
[the remark this answers is not in the document]
Comment (extrowerk) [reply media-gfx/povray/povray-3.7.0.8.recipe]: At least tell me, what does $libDir and B_SYSTEM_LIB_DIRECTORY results on x86, and why are they not ok here.

File: media-gfx/povray/povray-3.7.0.8.recipe [hunk 3]
```diff
@@ -77,14 +77,24 @@ BUILD()
 {
 	cd unix
 	./prebuild.sh
+	autoupdate
```
[the remark this answers is not in the document]
Comment (kenmays) [reply media-gfx/povray/povray-3.7.0.8.recipe]: **autoupdate** is a command component of the _autoconf_ package.

File: media-gfx/povray/povray-3.7.0.8.recipe (hunk 2 above)
[the remark this answers is not in the document]
Comment (kenmays) [reply media-gfx/povray/povray-3.7.0.8.recipe]: B_SYSTEM_LIB_DIRECTORY = **/system/lib** versus **/system/lib/x86** (_2nd_ arch).

[the remark this answers is not in the document]
Comment (extrowerk) [reply media-gfx/povray/povray-3.7.0.8.recipe]: Probably you want to remove the BoostLibLctr hack, and....

File: media-gfx/povray/povray-3.7.0.8.recipe [hunk 4]
```diff
@@ -77,14 +77,24 @@ BUILD()
...
+	KeepEXR=""
+
+	if [ $effectiveTargetArchitecture = x86_gcc2 ]; then
+		BoostLibLctr=$BoostLibLctr/x86
+		KeepEXR="--without-openexr"
+	fi
 
 	runConfigure ./configure \
 		COMPILED_BY="HaikuPorts" \
-		--with-boost-libdir=`finddir B_SYSTEM_LIB_DIRECTORY`
```
Comment (extrowerk) [inline media-gfx/povray/povray-3.7.0.8.recipe]: ...just use:
--with-boost-libdir="`finddir B_SYSTEM_LIB_DIRECTORY`/${secondaryArchSubDir}"

Comment (kenmays) [reply media-gfx/povray/povray-3.7.0.8.recipe]: Applied.

File: media-gfx/povray/povray-3.7.0.8.recipe (hunk 2 above)
[the remark this answers is not in the document]
Comment (kenmays) [reply media-gfx/povray/povray-3.7.0.8.recipe]: Applied.

**Pull request merged**